### PR TITLE
AWS AMI creation

### PR DESCRIPTION
**Pull request by reneab**
## Description

Infrastructure changes to create AMIs on AWS and ideally automate their creation based on release tags

## Type of Change

- [ ] 📚 Examples / docs / tutorials / dependencies update
- [ ] 🔧 Bug fix (non-breaking change which fixes an issue)
- [x] 🥂 Improvement (non-breaking change which improves an existing feature)
- [ ] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🔐 Security fix

## Checklist

- [ ] I've read the [`CODE_OF_CONDUCT.md`](https://github.com/Giskard-AI/ai-inspector/blob/master/CODE_OF_CONDUCT.md) document.
- [ ] I've read the [`CONTRIBUTING.md`](https://github.com/Giskard-AI/ai-inspector/blob/master/CONTRIBUTING.md) guide.
- [ ] I've updated the code style using `make codestyle`.
- [ ] I've written tests for all new methods and classes that I created.
- [ ] I've written the docstring in Google format for all the methods and classes that I used.

### CODE REVIEW

**Comment by andreybavt** [comment]
@reneab could you please also add systemd config file to git so that we keep track of it too?

**Comment by reneab** [comment]
> @reneab could you please also add systemd config file to git so that we keep track of it too?

You're right, it's done.
I also added a command there to prune images after push and up, it's better this way

**Review by andreybavt — APPROVED** [review]

File: scripts/infrastructure/giskard-docker-start.service [hunk 1]
```diff
@@ -10,14 +10,13 @@ WorkingDirectory=/home/ec2-user
...
-# change this to branch main before merging
-ExecStartPre=-/bin/bash -c "/bin/curl -f 'https://raw.githubusercontent.com/Giskard-AI/giskard/feat/ami/docker-compose.ami.yml' -o docker-compose.ami.yml"
+ExecStartPre=-/bin/bash -c "/bin/curl -f 'https://raw.githubusercontent.com/Giskard-AI/giskard/main/docker-compose.ami.yml' -o docker-compose.ami.yml"
 ExecStartPre=-/bin/bash -c "/bin/curl -f 'https://raw.githubusercontent.com/Giskard-AI/giskard/main/docker-compose.yml' -o docker-compose.yml"
 ExecStartPre=-/usr/local/bin/docker-compose pull --quiet
 ExecStart=/usr/local/bin/docker-compose -f docker-compose.yml -f docker-compose.ami.yml up -d
 
 ExecStartPost=/bin/bash -c "echo 'Pruning docker images...'"
-ExecStartPost=-docker image prune -f
+ExecStartPost=-/usr/local/bin/docker image prune -f
```
Comment (reneab) [inline scripts/infrastructure/giskard-docker-start.service:19]: Hello @andreybavt je viens de checker et c'est pas correct, le path to docker c'est /usr/bin par default (et le path to docker-compose est different car il est installe differement, pas par yum)
Si tu es ok je vais pusher ca directement sur main?

Comment (andreybavt) [reply scripts/infrastructure/giskard-docker-start.service:19]: Salut @reneab , oui, push le directement, merci!

Comment (reneab) [reply scripts/infrastructure/giskard-docker-start.service:19]: ok et je rajoute un petit script aussi d'installation de docker et docker-compose sur AWS Linux, ca manquait, sorry

Comment (reneab) [reply scripts/infrastructure/giskard-docker-start.service:19]: bon en fait j'ai cree une petit pull request c'est mieux 
https://github.com/Giskard-AI/giskard/pull/184

**Pull request merged**